migrations/63_issue_type_hex_color.up.sql

```diff
@@ -0,0 +1,19 @@
+UPDATE issue_type
+SET hex_color = '#D32F2F'
+WHERE issue_name = 'Product Bug';

```

Comment:
Use id or locator in WHERE instead of issue_name, for all insertion below, please.